warehouse/templates/email/trusted-publisher-added/body.html

```diff
@@ -28,7 +28,9 @@
   {% trans %}Publisher information{% endtrans %}:
   <ul>
     <li>{% trans %}Publisher name{% endtrans %}: {{ publisher_name }}</li>
-    <li>{% trans %}Publisher specification{% endtrans %}: {{ publisher_spec }}</li>
+    <li>{% trans %}Workflow{% endtrans %}: {{ publisher_workflow }}</li>
+    <li>{% trans %}Repository{% endtrans %}: {{ publisher_repository }}</li>
+    <li>{% trans %}Environment{% endtrans %}: {{ publisher_environment }}</li>
```

Comment:
Nit (here and below): `publisher_environment` will be `None` if not configured, so this will end up rendering an explicit "None".

We probably want to render it conditionally (skip it if it isn't present) or with a fallback, e.g. `"N/A"`.

Reply:
Resolved.